### ISSUE TITLE
TestUpdateESSecureSettings, invalid keystore entries

### ISSUE THREAD

**Issue by artemnikitin** [elastic/cloud-on-k8s#1089]
```go
09:56:42      --- FAIL: TestUpdateESSecureSettings/Secure_settings_should_eventually_be_set_in_all_nodes_keystore (300.00s)
09:56:42          require.go:794: 
09:56:42              	Error Trace:	testutils.go:42
09:56:42              	Error:      	Received unexpected error:
09:56:42              	            	invalid keystore entries. Expected: [file.setting1 file.setting2 key.without.prefix string.setting1 string.setting2]. Actual: [file.setting1 file.setting2]
09:56:42              	Test:       	TestUpdateESSecureSettings/Secure_settings_should_eventually_be_set_in_all_nodes_keystore
09:56:42  FAIL
```

**Comment by barkbay** [comment]
Looks like a duplicate of https://github.com/elastic/cloud-on-k8s/issues/733

**Comment by barkbay** [comment]
Closed as duplicate of #733
